app/file.php

```diff
@@ -1,2 +1,50 @@
 <?php
 
+declare(strict_types = 1);
+$root = dirname(__DIR__) . DIRECTORY_SEPARATOR;
+define('FILES_PATH', $root . 'files' . DIRECTORY_SEPARATOR);
```

Comment:
As a good practice, is not a good idea to declare global constants but is not wrong here. Just be mindful that it will expose information that maybe is not intended to be outside this class.